.github/workflows/sync-critical-path.yml

```diff
@@ -72,13 +72,6 @@ jobs:
           env: |
             CODE=${{ steps.sync-recovery-code.outputs.recovery-code }}
 
-      - name: Maestro Output
```

Comment:
you don’t need to remove this, it’s actually useful.

Reply:
Agree, but I prefer a clean revert for now, and I can bring them back on a new PR. I